lib/ws_servers/api/handlers/on_selected_algo_resume_remove.js

```diff
@@ -0,0 +1,55 @@
+const send = require('../../../util/ws/send')
+const validateParams = require('../../../util/ws/validate_params')
+const isAuthorized = require('../../../util/ws/is_authorized')
+const sendError = require('../../../util/ws/send_error')
+
+module.exports = async (server, session, msg) => {
+  const { d, algoDB } = server
+  const { AlgoOrder } = algoDB
+  const [, authToken, payload] = msg
+
+  const validRequest = validateParams(session, {
+    authToken: { type: 'string', v: authToken },
+    payload: { type: 'object', v: payload }
+  })
+
+  if (!validRequest) {
+    d('invalid request: selected_algo:resume_remove')
+    return
+  }
+
+  if (!isAuthorized(session, authToken)) {
+    return sendError(session, 'Unauthorized', ['unauthorized'])
+  }
+
+  const tasks = []
+
+  for (const [mode, { algoWorker }] of Object.entries(session.services)) {
+    if (algoWorker) {
```

Comment:
We should create algo worker if it does not exist, imo.

Reply:
Yes, good point. I've checked that only `ConnectionManager._startAlgoWorker` seems creating the **algoWorker**. Shall I use that to start the algo worker? @tarcisiozf @avsek477

Reply:
I don't think its this handler's responsibility to do so, because this is a session-related behavior. If the request passed the login check above it means that the session is established and authenticated.

Reply:
But another point to be made is that meta-reloaded is a connection-instability error, will the algo worker recover from such problem? if not we need find a way to detect and replace it.

Reply:
If the user logs in and never changes mode, and the network disconnection happens, when we send all AOs, there might be a case that BE returns the AOs of other mode as well. In that case, we never started the algo worker and trying to resume/cancel such AOs will always be neglected.
We do have reconnect logic, I think. But, please do check once @arifjmamun